### PR TITLE
Remove image preview for Disputes Evidence upload fields settings

### LINKED ISSUE #3853: Deposits evidence file preview shows an invalid image
### Describe the bug
Deposits evidence file preview is broken due to recent changes in image upload component with #3713

### To Reproduce
1. Go to 'Payments' -> 'Disputes'.
2. Open one of the disputes and click `Challenge dispute`.
3. Select 'Physical product' from the `Product type` dropdown.
4. Upload a `Custom signature` image and clarify the preview is broken.

### Screenshots
Screenshot 2022-02-24 at 10 23 27

### Expected behavior
Disputes evidence upload settings shouldn't have a preview of the uploaded file as it could be not an image file.

### PR DESCRIPTION
**Pull request by mordeth**
Fixes #3853

#### Changes proposed in this Pull Request
Disputes evidence upload field settings to display file name instead of preview as the uploaded files are not only images.

#### Testing instructions
- Go to 'Payments' -> 'Disputes'.
- Open one of the disputes and click Challenge dispute.
- Select 'Physical product' from the `Product type` dropdown.
- Upload a file to the `Custom signature` setting field.
- Confirm the preview of the file is not displayed and the file name is displayed instead.
--------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

### CODE REVIEW

File: client/disputes/evidence/file-upload.tsx [hunk 1]
```diff
@@ -71,24 +72,32 @@ export const FileUploadControl = ( {
...
-								fileName +
-								'?as_account=0'
-							}
-							style={ { maxWidth: 100 } }
-							alt={ fileName }
-						/>
-					)
+				) }
+
+				{ ! hasError && (
```
Comment (anu-rock) [inline client/disputes/evidence/file-upload.tsx:81]: The error and preview conditions have become a bit complex. Perhaps this part can be extracted as a sub-component?

File: client/card-readers/settings/file-upload.js [hunk 2]
```diff
@@ -130,6 +130,7 @@ const BrandingFileUpload = ( props ) => {
 				error={ error }
 				onFileChange={ handleFileChange }
 				onFileRemove={ handleFileRemove }
+				hasPreview={ true }
```
Comment (anu-rock) [inline client/card-readers/settings/file-upload.js:133]: How about `showPreview`?

**Review by anu-rock — APPROVED** [review]
Great work! Thanks for noticing the issue and fixing it rapidly 💨  
I've left a couple of minor comments.

I tested in Firefox.

Before:  
Screenshot 2022-02-25 at 2 58 17 PM

After:  
Screenshot 2022-02-25 at 2 57 36 PM

File: client/card-readers/settings/file-upload.js (hunk 2 above)
Comment (kalessil) [reply client/card-readers/settings/file-upload.js:133]: +1

File: changelog/fix-3853-deposits-evidence-upload-preview [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: tweak
+Comment: Fixing a feature that is not released yet.
```
Comment (kalessil) [inline changelog/fix-3853-deposits-evidence-upload-preview:3]: That's looks different from what we have in other files, can you ensure the format is correct, please?

File: client/disputes/evidence/file-upload.tsx (hunk 1 above)
Comment (kalessil) [reply client/disputes/evidence/file-upload.tsx:81]: Or a local variable?

**Review by kalessil — APPROVED** [review]
LGTM! Left a few comments on how to improve things a little bit further.

File: client/card-readers/settings/file-upload.js (hunk 2 above)
Comment (mordeth) [reply client/card-readers/settings/file-upload.js]: Updated.

File: client/disputes/evidence/file-upload.tsx (hunk 1 above)
Comment (mordeth) [reply client/disputes/evidence/file-upload.tsx]: I have moved the File Upload component to the `components` folder as it is not Disputes only feature anymore.

**Review by dpaun1985 — APPROVED** [review]
LGTM

File: changelog/fix-3853-deposits-evidence-upload-preview (hunk 3 above)
Comment (mordeth) [reply changelog/fix-3853-deposits-evidence-upload-preview:3]: It is correct. The `Comment` is used for internal comment only as we don't need a public changelog entry.

**Pull request merged**